c/include/libsbp/navigation.h

```diff
@@ -540,6 +540,117 @@ typedef struct SBP_ATTR_PACKED {
   u8 flags;      /**< Status flags */
 } msg_pos_llh_cov_t;
 
+typedef struct SBP_ATTR_PACKED {
+  float semi_major;  /**< The semi major axis of the estimated horizontal
```

Comment:
I know it's a pain to rebuild all the bindings, but I'd suggest we append a suffix for the unit, like `semi_major_m` and `orientation_deg` etc. :)

Reply:
re. the msg itself, if you prefer having the same field names as the original msg_po_llh, no worries.

Reply:
Regarding the units we cannot just add the suffix in the C++ code, since it's generated from the message definition. And we typically don't add the units to the field names. Maybe this is something that can be addressed in the code generator. But I guess this would break a lot of things and be backwards incompatible.